src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommandWorker.cs

```diff
@@ -27,18 +28,19 @@ internal sealed class GetDocumentCommandWorker
     private const string InvalidFilenameString = "..";
     private const string JsonExtension = ".json";
     private const string UnderscoreString = "_";
-    private static readonly char[] InvalidFilenameCharacters = Path.GetInvalidFileNameChars();
-    private static readonly Encoding UTF8EncodingWithoutBOM
+    private static readonly char[] _invalidFilenameCharacters = Path.GetInvalidFileNameChars();
```

Comment:
I usually capitalize constants.  Is that not the local convention?

Reply:
I believe because these are technically `static readonly`s and not `const`s the conventions on private variables are applying here.